BrainPortal/app/controllers/nh_invitations_controller.rb

```diff
@@ -132,6 +132,13 @@ def update #:nodoc:
     @nh_invitation.save
 
     flash[:notice] = "You have been added to project #{@nh_project.name}."
+
+    Message.send_message(@nh_invitation.sender,
```

Comment:
By the way, here, why not use @nh_project.name instead of @nh_imvitation.group.name ?

Reply:
ok that's easy. I experimented a bit to make cut and paste from cbrain which we often use for neurohub smoother ( you do not have to remember that groups became nh_projects, just add/remove nh). But can change.